app/scripts/modules/kubernetes/src/v2/serverGroupManager/details/details.controller.ts

```diff
@@ -86,13 +93,13 @@ class KubernetesServerGroupManagerDetailsController implements IController {
           namespace: this.serverGroupManager.namespace,
           account: this.serverGroupManager.account,
         },
-        revisions: () =>
-          this.serverGroupManager.serverGroups.map(sg => {
-            return {
-              name: sg.name,
-              revision: sg.moniker.sequence,
-            };
-          }),
+        revisions: () => {
```

Comment:
Should we remove the current active revision?

Reply:
The `const [, ...rest]` bit below pulls out the first revision (after sorting by the sequence).